core/src/main/java/ch/cyberduck/core/AbstractProtocol.java

```diff
@@ -407,6 +407,13 @@ public <T> T getFeature(final Class<T> type) {
         if(type == ComparisonService.class) {
             return (T) new DefaultComparisonService(this);
         }
+        if(type == PasswordStorePrefixService.class) {
```

Comment:
Implementation via protocol feature without configurable property for implementation forces sub-classing all used protocols, not full implementation of #15113. Re-introduce factory for `factory.passwordstoreprefixservice.class` (and second Windows one)?

Reply:
We have no use case where the implementation must be replaced for more than one protocol implementation?